sap/adt/function.py

```diff
@@ -208,3 +208,75 @@ def function_group_reference(self):
             self._reference = FunctionGroup.create_reference(self.connection, self._function_group_name, self.package)
 
         return self._reference
+
+
+class FunctionInclude(ADTObject):
+    """ABAP Function Group Include"""
+
+    OBJTYPE = ADTObjectType(
+        'FUGR/I',
+        'functions/groups/{groupname}/includes',
+        xmlns_adtcore_ancestor('finclude', 'http://www.sap.com/adt/functions/fincludes'),
+        ['application/vnd.sap.adt.functions.fincludes+xml',
+         'application/vnd.sap.adt.functions.fincludes.v2+xml'],
```

Comment:
Don't we use the reverse order in other objects? The newest version is the first item?

Reply:
The first item is used as the default value for the HTTP header Content-Type when sending request to ADT.